components/gcp-click-to-deploy/src/DeployForm.tsx

```diff
@@ -510,15 +560,29 @@ export default class DeployForm extends React.Component<any, DeployFormState> {
   /**
    * Returns a list of services that are needed but not enabled for the given project.
    */
-  private async _getServicesToEnable(project: string) {
-    const enabledServices = await Gapi.servicemanagement.list(project);
+  private async _getServicesToEnable(project: string, token: string) {
+    // const enabledServices = await Gapi.servicemanagement.list(project);
+    const consumerId = encodeURIComponent(`project:${project}`);
+    const enabledServices = await rp(
```

Comment:
What's the difference between this new code and the old `servicemanagement` object?
It's also good to keep all google APIs contained in the Gapi module, why not make these changes in there?